### PR TITLE
shorten test failure message in Developers hipchat room.

### PR DESCRIPTION
**Pull request by davidsbailey**

### CODE REVIEW

File: dashboard/test/ui/runner.rb [hunk 1]
```diff
@@ -299,11 +299,12 @@ def output_synopsis(output_text)
       link = "https://test-studio.code.org/ui_test/" + html_output_filename
       message += " <a href='#{link}'>☁ html output</a>"
     end
+    short_message = message
```
Comment (laurelfan) [inline dashboard/test/ui/runner.rb:302]: I don't get what this does, am I missing something?

File: dashboard/test/ui/runner.rb [hunk 2]
```diff
@@ -299,11 +299,12 @@ def output_synopsis(output_text)
...
 
     message += "<br/><i>command line: cucumber #{arguments + first_time_arguments}</i>"
     message += "<br/><i>rerun: ./runner.rb -c #{browser_name} -f #{feature} --html</i>"
     HipChat.log message, color: 'red'
-    HipChat.developers message, color: 'red' if CDO.hip_chat_logging
+    HipChat.developers short_message, color: 'red' if CDO.hip_chat_logging
```
Comment (davidsbailey) [inline dashboard/test/ui/runner.rb:306]: it prints a shorter thing to the developers room

Comment (laurelfan) [inline dashboard/test/ui/runner.rb]: what shorter thing? as far as I can tell it is the same thing (short_message = message)

**Comment by laurelfan** [comment]
Oh I see now, Was just looking at the changed lines not the whole file :p. LGTM. You can probably also remove the cucumber line altogether (that was mostly for debugging... the rerun line is the useful one).

**Pull request merged**